src/mercury/didcomm/Wrapper.ts

```diff
@@ -173,8 +173,8 @@ export class DIDCommWrapper implements DIDCommProtocol {
 
     return new Domain.AttachmentDescriptor(
       this.parseAttachmentDataToDomain(attachment.data),
-      attachment.id,
       attachment.media_type,
+      attachment.id,
```

Comment:
The solution works but create a ticket for @elribonazo to remove the work around logic

Reply:
Which workaround?